From c26239a8c67e1787a2243bd4fb9ea4349445f430 Mon Sep 17 00:00:00 2001
From: "guoyin.chen" <guoyin.chen@freescale.com>
Date: Fri, 27 Mar 2015 18:11:32 +0800
Subject: [PATCH] MA-6470 Update android_defconfig based on imx_v7_defconfig
 update in imx_3.14.y

Add config CONFIG_ARM_IMX7D_CPUFREQ, CONFIG_GPIO_MAX732X, CONFIG_MXC_IPU_V3_PRE,
and CONFIG_AD2802A

Signed-off-by: guoyin.chen <guoyin.chen@freescale.com>
---
 arch/arm/configs/imx_v7_android_defconfig | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/configs/imx_v7_android_defconfig b/arch/arm/configs/imx_v7_android_defconfig
index b83eefd..2a8fdf0 100644
--- a/arch/arm/configs/imx_v7_android_defconfig
+++ b/arch/arm/configs/imx_v7_android_defconfig
@@ -54,6 +54,7 @@ CONFIG_CPU_FREQ_GOV_USERSPACE=y
 CONFIG_CPU_FREQ_GOV_ONDEMAND=y
 CONFIG_CPU_FREQ_GOV_CONSERVATIVE=y
 CONFIG_ARM_IMX6Q_CPUFREQ=y
+CONFIG_ARM_IMX7D_CPUFREQ=y
 CONFIG_CPU_IDLE=y
 CONFIG_VFP=y
 CONFIG_NEON=y
@@ -313,6 +314,7 @@ CONFIG_SPI=y
 CONFIG_SPI_GPIO=y
 CONFIG_SPI_IMX=y
 CONFIG_GPIO_SYSFS=y
+CONFIG_GPIO_MAX732X=y
 CONFIG_GPIO_74X164=y
 CONFIG_SABRESD_MAX8903=y
 CONFIG_IMX6_USB_CHARGER=y
@@ -491,6 +493,7 @@ CONFIG_MMC_SDHCI_ESDHC_IMX=y
 CONFIG_MXC_IPU=y
 CONFIG_MXC_GPU_VIV=y
 CONFIG_GPU_LOW_MEMORY_KILLER=y
+CONFIG_MXC_IPU_V3_PRE=y
 CONFIG_MXC_MIPI_CSI2=y
 CONFIG_MXC_MLB150=y
 CONFIG_LEDS_GPIO=y
@@ -525,6 +528,7 @@ CONFIG_ION_MXC=y
 # CONFIG_IOMMU_SUPPORT is not set
 CONFIG_IIO=y
 CONFIG_VF610_ADC=y
+CONFIG_AD2802A=y
 CONFIG_PWM=y
 CONFIG_PWM_IMX=y
 CONFIG_EXT2_FS=y
-- 
1.8.0

